packages/augur-ui/src/modules/market/components/market-trade-history/market-trade-history.styles.less

```diff
@@ -9,164 +9,122 @@
-  max-height: inherit;
-  padding: 18px 24px;
-  width: 100%;
-}
-
-.MarketTradeHistory__table {
-  display: table;
-  position: sticky;
-  top: 0;
-  width: 100%;
-  z-index: @above-all-content;
-}
-
-.MarketTradeHistory__table-header {
-  background-color: @color-theme-primary;
-  color: @color-neutral;
-  display: flex;
-  font-size: @font-size-small;
-  justify-content: space-between;
-  padding: 0.5625rem 1.5rem 0.5625rem 1.875rem;
-  width: 100%;
-
-  > li {
-    &:first-child {
-      padding-right: 0.8rem;
-    }
 
-    &:nth-child(2) {
-      padding-right: 2rem;
-    }
+  > div {
+    display: flex;
+    flex-direction: column;
+    flex-grow: 1;
+    overflow: overlay;
+    overflow-y: auto;
   }
 }
 
-.MarketTradeHistory__table__trade__data {
-  background-color: @color-tertiary-background;
-  color: @color-secondary-text;
-  display: flex;
-  font-size: @size-11;
-  padding-bottom: 0.25rem;
-  position: relative;
-  width: 100%;
-
-  > li {
-    font-family: 'Roboto Mono', monospace;
-    padding-bottom: 4px;
-    padding-top: 4px;
-
-    &:first-child {
-      text-align: left;
-    }
+.TradeHistoryTable {
+  > ul:first-of-type {
+    background-color: fade(@color-primary-alternate, 70%);
+    color: @color-shadow-text;
+    display: flex;
+    font-size: @size-10;
+    justify-content: center;
+    width: 100%;
+    position: sticky;
+    top: 0;
+    z-index: @above-all-content;
+
+    > li {
+      display: table-cell;
+      padding-bottom: @size-6;
+      padding-top: @size-6;
+      text-align: right;
 
-    &:nth-child(2) {
-      flex: 1;
-      justify-content: right;
-      padding-left: 0.625rem;
-    }
+      &:first-child {
+        padding-left: @size-32;
+        text-align: left;
+        text-transform: uppercase;
+      }
 
-    &:nth-child(3) {
-      flex: 1;
-      justify-content: right;
-      padding-left: 1rem;
-    }
+      &:nth-of-type(2) {
+        margin: 0 @size-8;
+      }
 
-    &:last-child {
-      color: @color-neutral;
-      padding-right: 25px;
-      text-align: right;
+      &:last-child {
+        padding-right: @size-24;
+      }
     }
   }
-}
-
-.MarketTradeHistory__table__header__day {
-  background-color: fade(@color-primary-alternate, 70%);
-  color: @color-shadow-text;
-  display: flex;
-  font-size: @font-size-extra-small;
-  justify-content: center;
-  width: 100%;
-
-  > li {
-    display: table-cell;
-    padding-bottom: 6px;
-    padding-top: 6px;
-    text-align: right;
-
-    &:first-child {
-      padding-left: 32px;
-      text-align: left;
-      text-transform: uppercase;
-    }
 
-    &:nth-of-type(2) {
-      margin: 0 0.5rem;
-    }
-
-    &:last-child {
-      padding-right: 25px;
+  > ul:not(:first-of-type) {
+    background-color: @color-tertiary-background;
+    color: @color-secondary-text;
+    display: flex;
+    font-size: @size-11;
+    padding-bottom: @size-4;
+    position: relative;
+    width: 100%;
+
+    > li {
+      font-family: @roboto-mono;
```

Comment:
use the `.mono-whateversizefontthisshouldbe` here instead of directly editing font-family